recipes/klayout/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set version = "0.27.12" %}
+
+package:
+  name: klayout
+  version: {{ version }}
+
+source:
+  url: https://github.com/KLayout/klayout/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 6ef5e6ae5ea387618663949a1efa105406aeb36db2d18d89eff0345955aae979
+  patches:
+    - force-python-dynamic-link.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+  ignore_run_exports:
+    - qt
+  missing_dso_whitelist:  
+    - "*/libnet_tracer.so.0"
+    - "*/liblefdef.so.0"
+    - "*/libpcb.so.0"
+
+requirements:
+  build:
+    - make
+    - {{ compiler('cxx') }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ compiler('c') }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxext') }}  # [linux]
+    - {{ cdt('libxfixes') }}  # [linux]
+    - {{ cdt('libxau') }}  # [linux]
+    - {{ cdt('libxcb') }}  # [linux]
+    - {{ cdt('expat') }}  # [linux]
+    - {{ cdt('libpng') }}  # [linux]
+
+  host:
+    - ruby >2
+    - qt >=5.0,<6.0
```

Comment:
The host section is for libraries that are linked-to and must match the target platform when cross-compiling.

It looks like you link against libpython, so python should be listed in the host requirements. Otherwise, the runtime python version might not match the version that you package links against.

Other libraries which I suspect should be moved to this section are zlib and libexpat.